config/default.yml

```diff
@@ -31,6 +31,7 @@ Rails/ActiveRecordAliases:
                   Use `update!` instead of `update_attributes!`.
   Enabled: true
   VersionAdded: '0.53'
+  Safe: false
```

Comment:
Could you mark it with `SafeAutoCorrect: false` instead of `Safe: false` when auto-correct is not safe?